addons/mail/static/src/emoji_picker/emoji_picker.js

```diff
@@ -202,16 +229,13 @@ export class EmojiPicker extends Component {
             }
             case "ArrowDown": {
                 const newIndex = this.state.activeEmojiIndex + EMOJI_PER_ROW;
-                if (newIndex <= this.getEmojis().length - 1) {
+                if (newIndex < this.itemsAmount) {
                     this.state.activeEmojiIndex = newIndex;
                 }
                 break;
             }
             case "ArrowRight": {
-                const newIndex = Math.min(
-                    this.state.activeEmojiIndex + 1,
-                    this.getEmojis().length - 1
-                );
+                const newIndex = Math.min(this.state.activeEmojiIndex + 1, this.itemsAmount - 1);
                 if (newIndex !== this.state.activeEmojiIndex) {
                     this.state.activeEmojiIndex = newIndex;
                     ev.preventDefault();
```

Comment:
I feel like it could be a bit simpler:
```suggestion
                if (this.state.activeEmojiIndex + 1 === this.itemsAmount) {
                    break;
                }
                this.state.activeEmojiIndex++;
                ev.preventDefault();
```